libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/DropInNavigationView.kt

```diff
@@ -67,6 +68,10 @@ class DropInNavigationView @JvmOverloads constructor(
         navigationContext.uiBinders = navigationUIBinders
     }
 
+    fun customize(options: MapboxRouteLineOptions) {
```

Comment:
nit: Will you put a comment here? Pretending it's a public api so we have less docs to write